Bottom: 8699f66a1eab8d7ef7e269eb0c638ac3636fa0d7
Top:    a4660000ac48ec7a130da623e35f929e45d759e3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:29:54 -0500

Move ansible role vars to role defaults

This allows other parts of the playbook i.e. host vars, command line the
ability to override these defaults when necessary.


---
diff --git a/roles/docker-openssl/vars/main.yml b/roles/docker-openssl/defaults/main.yml
similarity index 100%
rename from roles/docker-openssl/vars/main.yml
rename to roles/docker-openssl/defaults/main.yml
